### PR TITLE
handle return for integer module

### PR DESCRIPTION
**Pull request by norhh**
will fix a part of the bug in #935 .

### CODE REVIEW

File: mythril/analysis/modules/integer.py [hunk 1]
```diff
@@ -215,6 +218,25 @@ def _handle_jumpi(state):
                 )
             )
 
+    @staticmethod
+    def _handle_return(state: GlobalState) -> None:
```
Comment (JoranHonig) [inline mythril/analysis/modules/integer.py]: Can you add a comment detailing what this function adds.

File: mythril/analysis/modules/integer.py [hunk 2]
```diff
@@ -215,6 +218,30 @@ def _handle_jumpi(state):
...
+        locations in the memory returned by RETURN opcode.
+        :param state: The Global State
+        """
+        stack = state.mstate.stack
+        try:
+            offset, length = get_concrete_int(stack[-1]), get_concrete_int(stack[-2])
+        except TypeError:
+            return
+        for element in state.mstate.memory[offset : offset + length]:
+            if isinstance(element, Expression):
```
Comment (JoranHonig) [inline mythril/analysis/modules/integer.py]:
```suggestion
            if not isinstance(element, Expression):
                return
```

Comment (JoranHonig) [reply mythril/analysis/modules/integer.py]: To reduce nesting

File: mythril/analysis/modules/integer.py [hunk 3]
```diff
@@ -215,6 +218,30 @@ def _handle_jumpi(state):
...
+                for annotation in element.annotations:
+                    if isinstance(annotation, OverUnderflowAnnotation):
```
Comment (JoranHonig) [inline mythril/analysis/modules/integer.py]: There is a specific getter function, which you can pass the type of annotation you want too receive, could you use that here?

Comment (norhh) [reply mythril/analysis/modules/integer.py]: Isn't that for state annotations?

Comment (JoranHonig) [reply mythril/analysis/modules/integer.py]: Right, nvm. We should add that for these annotations too later 😅

File: mythril/analysis/modules/integer.py [hunk 4]
```diff
@@ -215,6 +218,31 @@ def _handle_jumpi(state):
...
+        :param state: The Global State
+        """
+        stack = state.mstate.stack
+        try:
+            offset, length = get_concrete_int(stack[-1]), get_concrete_int(stack[-2])
+        except TypeError:
+            return
+        for element in state.mstate.memory[offset : offset + length]:
+            if not isinstance(element, Expression):
+                return
```
Comment (JoranHonig) [inline mythril/analysis/modules/integer.py]:
```suggestion
                continue
```

**Pull request merged**